core/ui/src/components/settings/CreateOrEditHttpRouteModal.vue

```diff
@@ -274,6 +274,23 @@ export default {
       this.clearErrors();
       this.$emit("hide");
     },
+    validateInput(prop) {
+      const allowedCharsRegex = /^[a-zA-Z0-9\-_]+$/;
+      let badChars = prop.match(/[^a-zA-Z0-9\-_]/g);
+      if (badChars && !allowedCharsRegex.test(prop)) {
+        // replace space with _(space) for better readability
+        badChars = badChars.map(char => char === ' ' ? '_(space)' : char);
```

Comment:
```suggestion
        badChars = badChars.map(char => char === ' ' ? '(space)' : char);
```
I would remove that underscore before `(space)` for clarity

Reply:
@DavidePrincipi is right, it's probably better to simply show "Allowed characters: A-Z, a-z, 0-9, _ and -"